scss/_global.scss

```diff
@@ -180,3 +180,7 @@ code {
     background-color: unset;
   }
 }
+
+.login-form button[type='submit'] {
```

Comment:
@Simon-Laux maybe move this to a better place - I am not so involved in css structure

Reply:
currently login styles seem to be still in scss

Reply:
src/renderer/components/Login-Styles.js

Reply:
or you can already create a new scss file like scss/login/login-styles.scss
and don't forget to add the file to manifest.scss when you do that.
(you don't need to move everything there, only creating it for this class if fine for now)